### ISSUE TITLE
Allow to "include" files not below the folder where Cargo.toml resides.

### ISSUE THREAD

**Issue by andrewdavidmackenzie** [axodotdev/cargo-dist#194]
My project builds a number of cross-platfrom wasm files I would like to distribute with my package (each target will have a native, target specific runtime to run them).

They get built into out_dir or target by rustc.

I would like to include them via a glob rooted outside package root (e.g. ../target/..)

Others may want to include something from another folder, including os folders etc?

**Comment by Gankra** [comment]
Agreed this should work, even though it's a big reproducibility footgun to have stuff "outside" the cargo workspace. The file include code needs a bunch of extra feature work that I'm hoping to delegate to https://crates.io/crates/axoasset but not 100% sure if that makes sense.
